Algorithm/Number_theory/readme.md

```diff
@@ -34,5 +35,9 @@
 
 <hr>
 
-## Questions :
 
+# Questions related to Number Theory
+ 
+ 1.Factorial of a number:[Java](/Code/Java/factorial.java)
```

Comment:
why have you added this code readme here?

Reply:
I just referred the link of the code here

Reply:
Delete it or not